go.mod

```diff
@@ -190,4 +190,7 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 )
 
-replace antrea.io/ofnet v0.6.0 => github.com/wenyingd/ofnet v0.0.0-20220817031400-cb451467adc1
+replace (
+	antrea.io/libOpenflow v0.8.0 => github.com/hongliangl/libOpenflow v0.0.0-20220826044458-d89f3d570e60
```

Comment:
Remember to use the lib in group antrea.io after the depenedent changes are merged.

Reply:
Thanks, will do.

Reply:
Done